content/influxdb/v2.0/reference/cli/influx/task/retry-failed.md

```diff
@@ -0,0 +1,92 @@
+---
+title: influx task retry-failed
+description: The `influx task retry-failed` command retries failed InfluxDB a task runs.
+menu:
+  influxdb_2_0_ref:
+    name: influx task retry-failed
+    parent: influx task
+weight: 201
+---
+
+The `influx task retry-failed` command retries failed InfluxDB a task runs.
+
+## Usage
+```
+influx task retry-failed [flags]
+```
+
+## Flags
+| Flag |                   | Description                                                               | Input type  | {{< cli/mapped >}}    |
+|:---- |:---               |:-----------                                                               |:----------: |:------------------    |
+|      | `--after`         | Retry task runs that occurred after this time (RFC3339 timestamp)         | string      |                       |
+| `-c` | `--active-config` | CLI configuration to use for command                                      | string      |                       |
+|      | `--configs-path`  | Path to `influx` CLI configurations (default `~/.influxdbv2/configs`)     | string      |`INFLUX_CONFIGS_PATH`  |
+|      | `--before`        | Retry task runs that occurred before this time (RFC3339 timestamp)        | string      |                       |
```

Comment:
```suggestion
|      | `--before`        | Retry task runs that occur before this time (RFC3339 timestamp)        | string      |                       |
```

Reply:
These failed task runs have failed have already occurred. There is only past context for this command. It can only retry task runs (executions) that have already happened.